ui/app/templates/components/secret-create-or-update.hbs

```diff
@@ -26,12 +26,7 @@
         />
       </p>
       {{#if (get this.validationMessages "path")}}
-        <AlertInline
```

Comment:
<img width="1094" alt="Screenshot 2023-11-30 at 11 31 12 AM" src="https://github.com/hashicorp/vault/assets/68122737/b90683b2-2aa7-4a77-80a7-4dbe72487404">

Reply:
Should path be capitalized? Just checked the current look on kv and it's "Path can't be blank."

Reply:
I realize this is outside the HDS scope as the message is coming from the model.

Reply:
We could use the capitalize template helper.

Reply:
👍 
<img width="347" alt="Screenshot 2023-11-30 at 4 18 43 PM" 
src="https://github.com/hashicorp/vault/assets/68122737/a8d8aea4-b066-420a-93a9-c9cff7c0c952">